prov/shm/src/smr_ep.c

```diff
@@ -1619,7 +1619,21 @@ static int smr_ep_srx_context(struct smr_domain *domain, size_t rx_size,
 	smr_init_queue(&srx->unexp_msg_queue, smr_match_msg);
 	smr_init_queue(&srx->unexp_tagged_queue, smr_match_tagged);
 
-	srx->recv_fs = smr_recv_fs_create(rx_size, NULL, NULL);
+	ret = ofi_bufpool_create(&srx->rx_pool, sizeof(struct smr_rx_entry),
+					16, 0, rx_size, OFI_BUFPOOL_NO_TRACK);
```

Comment:
Why 16 byte alignment?

Reply:
I followed how shm created the cmd_ctx bufpool https://github.com/ofiwg/libfabric/pull/8882/files, which has similar size with rx_entry.